columbo/_cli.py

```diff
@@ -178,10 +178,10 @@ def _update_answers_validate(
     value = cast(str, cli_values.get(question.name))
     if value is None:
         value = to_value(question.default, answers, str)
-    error = question.validate(value, answers)
-    if error:
+    result = question._validate_and_convert(value, answers)
+    if not result.valid:
```

Comment:
We call `_validate_and_convert` because we want to make sure that the object returned is a new validation response (either a `ValidationFailure` or `ValidationSuccess`).